backend/src/modules/player-profiles/core/application/command/CreateGoogleUserProfileCommandHandler.ts

```diff
@@ -0,0 +1,17 @@
+import { CommandHandler } from '../../../../../shared/core/application/command/CommandHandler';
+import { CreateGoogleUserProfile } from './CreateGoogleUserProfile';
+import { CommandResult } from '../../../../../shared/core/application/command/CommandResult';
+import { DomainEventPublisher } from '../../../../../shared/core/application/event/DomainEventBus';
+import { createUserProfile } from '../../domain/GoogleUserProfile';
+import { GoogleUserProfilesRepository } from '../GoogleUserProfilesRepository';
+
+export class CreateGoogleUserProfileCommandHandler implements CommandHandler<CreateGoogleUserProfile> {
```

Comment:
Wyglada spoko. Ewentualnie tutaj wlasnie powinien pojsc event. Wtedy mozesz wyszukac np. przydzielic na podstawie maila id z google do konkretnego usera.

Reply:
W sumie to mi to sie wydaje tak: Moduł, który nasluchuje na eventy np:
GoogleUserRegistered
UserRegistered 

czy cos takiego. 
Jesli email gdzies jest ten sam, to wtedy matchuje. W GoogleUserProfile zapisuje id usera w aplikacji (to nie Googlowe). Co o tym myslisz?